samples/samples.props

```diff
@@ -1,20 +1,16 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ImportGroup Label="PropertySheets" />
-  <PropertyGroup Label="UserMacros">
-    <EDK_PATH>$(SolutionDir)..\edk2</EDK_PATH>
-  </PropertyGroup>
   <PropertyGroup>
     <GenerateManifest>
```

Comment:
Hmm, the diff is still showing <GenerateManifest> having a second </GenerateManifest> underneath. There are two commits but they only touch the readme file and another props file. Did you forget to push?